Automattic/vip-go-mu-plugins
# SDI: add instant_search field to Jetpack object

## Description
We currently have no way to query if Instant Search has been enabled via SDI. This piggybacks onto the existing `is_instant_search()` logic at https://github.com/Automattic/jetpack/blob/124ee2d27b11a93735a8fce9c7a7e6296e920b7e/projects/plugins/jetpack/modules/search/class-jetpack-search-options.php#L55-L64
## Changelog Description

### Site Details Index: Add new field instant_search to Jetpack object

Uses logic from Jetpack_Search_Options::is_instant_search() to get the data.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test
1) Use a site with Instant Search enabled (e.g. 4327)
2) Check if the option returns the correct value
